docs/platform-integration/essentials/geolocation.md

```diff
@@ -1,136 +1,163 @@
-```
+  ```xml
+  <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
+  <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+  <uses-feature android:name="android.hardware.location" android:required="false" />
+  <uses-feature android:name="android.hardware.location.gps" android:required="false" />
+  <uses-feature android:name="android.hardware.location.network" android:required="false" />
+  ```
 
-Or directly into your **AndroidManifest.xml**:
+  If your application is targeting Android 10 - Q (API Level 29 or higher) and is requesting **LocationAlways**, you must also add this permission request:
 
-```xml
-<uses-permission android:name="android.permission.ACCESS_BACKGROUND_LOCATION" />
-```
+  ```xml
+  <uses-permission android:name="android.permission.ACCESS_BACKGROUND_LOCATION" />
+  ```
 
-If it recommended to read [Android documentation on background location updates](https://developer.android.com/training/location/permissions) as there are many restrictions that need to be considered.
+  \- or -
 
-[!INCLUDE [android-permissions](includes/android-permissions.md)]
+- Use the Android project properties:
+  
+  Right-click on the Android project and open the project's properties. Under _Android Manifest_ find the **Required permissions:** area and check the **ACCESS_COARSE_LOCATION** and **ACCESS_FINE_LOCATION** permissions. This will automatically update the _AndroidManifest.xml_ file.
 
-# [iOS](#tab/ios)
+> [!TIP]
+> Be sure to read the [Android documentation on background location updates](https://developer.android.com/training/location/permissions), as there are many restrictions that need to be considered.
 
-Your app's **Info.plist** must contain the `NSLocationWhenInUseUsageDescription` key in order to access the device’s location.
+[!INCLUDE [android-permissions](includes/android-permissions.md)]
 
-Open the plist editor and add the **Privacy - Location When In Use Usage Description** property and fill in a value to display to the user.
+# [iOS](#tab/ios)
 
-Or manually edit the file and add the following and update the rationale:
+In your _Info.plist_ file, add the following keys:
 
 ```xml
 <key>NSLocationWhenInUseUsageDescription</key>
 <string>Fill in a reason why your app needs access to location.</string>
 ```
 
+The `<string>` element is the reason the app is requesting access to location information. This text is shown to the user.
+
+An alternative to editing the _Info.plist_ file directly is opening the plist editor. In the editor you can add the **Privacy - Location When In Use Usage Description** property, and fill in a value to display to the user.
+
 # [Windows](#tab/windows)
 
-You must set the `Location` permission for the application. This can be done by opening the **Package.appxmanifest** and selecting the **Capabilities** tab and checking **Location**.
+In the _Package.appxmanifest_, under **Capabilities**, ensure that the **Location** capability is checked.
```

Comment:
Think this is worth a todo to double check nearer the time. Wondering if any location support is included in WinUI 3?